test/HostActivationTests/GivenThatICareAboutStandaloneAppActivation.cs

```diff
@@ -107,6 +107,35 @@ public void Running_Publish_Output_Standalone_EXE_with_Unbound_AppHost_Fails()
         }
 
         [Fact]
+        public void Running_Publish_Output_Standalone_EXE_By_Renaming_dotnet_exe_Fails()
+        {
+            var fixture = PreviouslyPublishedAndRestoredStandaloneTestProjectFixture
+                .Copy();
+
+            var appExe = fixture.TestProject.AppExe;
+
+            string hostExeName = $"dotnet{Constants.ExeSuffix}";
+            string builtHost = Path.Combine(RepoDirectories.HostArtifacts, hostExeName);
+            File.Copy(builtHost, appExe, true);
+
+            int exitCode = Command.Create(appExe)
+                .CaptureStdErr()
+                .CaptureStdOut()
+                .Execute(fExpectedToFail:true)
```

Comment:
Yes, that is a good idea so that we are not affected by false negative scenarios.